src/library/models.py

```diff
@@ -651,13 +651,14 @@ def latest_for_user(cls, user: ClusiveUser, max=10):
         return Paradata.objects.filter(user=user, last_view__isnull=False).order_by('-last_view')
 
     @classmethod
-    def get_reading_data(cls, period: Period, days=0, sort='name', username=None):
+    def get_reading_data(cls, period: Period, days=0, sort='name', books_sort='title', username=None):
```

Comment:
I'm worried that the change to the default sort of books here will affect how they are displayed on the dashboard panel that also uses this method.  If so, should be easily fixed by passing in the correct sort from the dashboard code.

Reply:
@bgoldowsky the dashboard doesn't offer the sorting function for individual books read by a user. Books on the dashboard are displayed in a random order at this point. The sort on the dashboard is at the user level on the total reading time of all books and total number of books that have been read. With adding the default sort of books, books on the dashboard will now be displayed in an alphabetical order of books.

If you are concerned about sorting books on book titles by default, I could change to the default to `None`. Please let me know what you think. Thanks.

Reply:
Current dashboard doesn't offer a choice of sorts, but it does include code to sort the books in descending order of time spent.  From development branch, library/models.py, line 702:
   `# Sort book entries by time`
   `entry['books'].sort(reverse=True, key=lambda p: p['hours'])`

Reply:
@bgoldowsky, thanks for pointing out the line. If sorting by hours is the default sorting order on the dashboard, and it's a duplicate with the new `book_sort` functionality. Do you think if it makes sense to set the default `book_sort` value to `time` instead of `title`? The line you pointed out can then be removed?

Reply:
@bgoldowsky after a chat with @klown, I have a better understanding of this comment. The default `books_sort` value remains `title` for `get_reading_data()` and `get_truncated_reading_data()`. But in the dashboard code, `books_sort="time"` is explicitly passed to `get_truncated_reading_data()`. Here's [an example of changed code](https://github.com/cindyli/clusive/blob/feature/CSL-2101-reading-details/src/pages/views.py#L213). The line 702 in the library/models.py - `entry['books'].sort(reverse=True, key=lambda p: p['hours'])` - is no longer needed.

Hope this addresses your concern. Please let me know if it doesn't. Thanks.